code/_globalvars/lists/wiremod.dm

```diff
@@ -7,13 +7,15 @@ GLOBAL_LIST_INIT(wiremod_basic_types, list(
 	PORT_TYPE_LIST,
 	PORT_TYPE_TABLE,
 	PORT_TYPE_ATOM,
+	PORT_TYPE_DATUM,
```

Comment:
wiremod_admin_basic_types should inherit what is in wiremod_basic_types. I also don't want any usecase with this to have to check if it is an admin circuit whenever using GLOB.wiremod_basic_types.

Reply:
How would you suggest making the datum datatype visible for admin circuits?

Reply:
Don't know, that implementation is up to you. I had a suggestion but figured it'd be too finnicky and lead to further problems down the line.

You are better off (for now) just not having PORT_TYPE_DATUM in the wiremod_basic_types list until better support can be added for circuit admin tooling.

Reply:
So I guess that means PORT_TYPE_DATUM won't be available for circuit variables and module ports on admin circuits.

Reply:
they wouldn't work for module ports anyways because you have to modify it whilst it's out of the circuit.

That's why the admin tools need slightly more development before we can introduce admin-only types.